### ISSUE TITLE
Why rx.js included in the bundle?

### ISSUE THREAD

**Issue by MariMax** [swimlane/ngx-datatable#1208]
**I'm submitting a ...** 
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**
when I include `NgxDatatableModule` into my lazy-loaded component, bundle size grows up 800kb, because rx.js included in your build

**Expected behavior**
I don't want such huge dependencies in the library, your code probably should be around 200kb - 300kb

**Reproduction of the problem**
probably will make later

**What is the motivation / use case for changing the behavior?**
I care about my users(they suffer enough) and don't want rx.js included twice in my bundle and in lazy-loaded bundle

**Please tell us about your environment:**
any

* **Table version:** 11.1.5

* **Angular version:** 5.1.1

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
any
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
any

**Comment by amcdnl** [comment]
Oh this should not be happening! This is a regression, fix coming!

**Comment by MariMax** [comment]
I guess [this is](https://github.com/swimlane/ngx-datatable/blob/master/src/datatable.module.ts#L3) the main reason, you use `Observable.fromEvent` in 3 directives
- [resizable](https://github.com/swimlane/ngx-datatable/blob/master/src/directives/resizeable.directive.ts#L63-L67)
- [long-press](https://github.com/swimlane/ngx-datatable/blob/master/src/directives/long-press.directive.ts#L53-L64)
- [draggable](https://github.com/swimlane/ngx-datatable/blob/master/src/directives/draggable.directive.ts#L73)

seems should be easy to replace with HostListener, it reduces the bundle size

**Comment by amcdnl** [comment]
fixed in 11.1.6
